cmd/device-worker/main.go

```diff
@@ -13,14 +15,15 @@ import (
 	"github.com/jakub-dzon/k4e-device-worker/internal/server"
 	workload2 "github.com/jakub-dzon/k4e-device-worker/internal/workload"
 
+	"log"
 	"net"
 	"os"
 	"path"
 	"time"
 
-	"git.sr.ht/~spc/go-log"
-
 	pb "github.com/redhatinsights/yggdrasil/protocol"
+	"go.uber.org/zap"
```

Comment:
What is the advantage to use this logger?

Reply:
I thought about using zap to allow a homogeneous log management for `k4e-operator` and `k4e-device-worker`.
Using different logs implies a different interface for the user in terms of verbosity levels.
Also, if I'm not wrong, go-log doesn't implement the logr.Logger interface, so we don't have the `WithValues` method to easily add context information to each log message in each involved project.
What do you think about that?

Reply:
I'd be interesting to learn how does it affect the output log file.
The output goes to the file which is created by yggd iiuc, and it should contain the output for yggdrasil and its workers.
Will the file have a mixed format and verbosity ?

Reply:
Without my PR:
if `yggd` is started with `--log-level error` and `k4e-worker` with default log level (`info`) then the output will be something like
```
[yggd] 2021/12/13 18:07:48 [/usr/local/libexec/yggdrasil/device-worker] time="2021-12-13T18:07:48+01:00" level=info msg=......
[yggd] 2021/12/13 18:07:48 [/usr/local/libexec/yggdrasil/device-worker] time="2021-12-13T18:07:48+01:00" level=warning msg= .......
```
so yggd prints his own error logs + k4e-worker all logs (info, warn and error).
In this case there is a mix in term of verbosity.

with my PR there is also a mix in term of format

```[yggd] 2021/12/13 17:49:12 listening on socket: @yggd-dispatcher-nyfcMI
[yggd] 2021/12/13 17:49:12 [/usr/local/libexec/yggdrasil/device-worker] {"level":"info","ts":1639414152.9651952,"logger":"device-worker","caller":"device-worker/main.go:65","msg":"....}
[yggd] 2021/12/13 17:49:12 worker registered: {pid:146390 handler:device addr:@ygg-device-YkeGUn features:map[] detachedContent:false}
[yggd] 2021/12/13 17:49:12 [/usr/local/libexec/yggdrasil/device-worker] {"level":"info","ts":1639414152.966989,"logger":"device-worker","caller":"device-worker/main.go:102","msg":"......"}
```
Did I address properly your doubt?

So probably it's better to keep log management homogeneous between `yggd` and `k4e-device-worker`. Right?

Reply:
yes, that was exactly my concern. since the entire output goes into the same log file, I'd prefer it to be the same format.